### ISSUE TITLE
What is the roadmap of the polymer-dart?

### ISSUE THREAD

**Issue by joo** [dart-lang/polymer-dart#717]
A three-year project must have a milestone！

**Comment by joo** [comment]
It's been four years since the WEB UI.

**Comment by Skeec** [comment]
Doesn't seem like polymer-dart is actively developed on right?
Since the new web spec is out and polymer is introducing polymer-2.0 getting way more modular we could do quit fast porting it right?

**Comment by dam0vm3nt** [comment]
let'see what they tell us at the next dartsummit, otherwise you can take a look at https://pub.dartlang.org/packages/polymerize

**Comment by Skeec** [comment]
Yeah thanks

**Comment by joo** [comment]
+dam0vm3nt, thanks!

**Comment by MortalCatalyst** [comment]
Dart summit featured preview angular components https://m.youtube.com/watch?v=vAUUOwBJetg

**Comment by tmst** [comment]
@MortalCatalyst Thanks for the signpost. I think I found the trail again.

**Comment by dam0vm3nt** [comment]
yeah. very interesting. but they can't be compared in any way to www.webcomponents.org right now. Maybe in the future they will. But I fear that the "lock-in" strategy of going all dart is not paying.
